packages/lesswrong/components/shortform/ShortformSubmitForm.tsx

```diff
@@ -36,25 +35,11 @@ const styles = (theme: ThemeType): JssStyles => ({
     padding: 0,
     border: "none",
   },
-  quickTakesEditor: {
-    background: "transparent",
-    padding: "10px 20px",
-  },
-  quickTakesTags: {
-    padding: "0 20px",
-  },
-  quickTakesButton: {
-    marginTop: 20,
-    padding: 20,
-    borderTop: `1px solid ${theme.palette.grey[300]}`,
-  },
 });
 
 const ShortformSubmitForm = ({
   successCallback,
   cancelCallback,
-  prefilledProps,
-  noDefaultStyles,
```

Comment:
I had forgotten that this existed, but these are used on the core topic page, where you can create a new quick take that is supposed to be on the given topic. It looks like this is broken on live as well so it doesn't need to be fixed in this PR, but maybe we should leave this props for now.

<img width="674" alt="Screen Shot 2024-04-17 at 5 40 28 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/d31dab14-4343-4fe6-9af2-ac0cd5f95845">